build.sh

```diff
@@ -124,6 +126,7 @@ build_protobuf() {
 		_log_info "creating \"${_n%.proto}.pb.go\""
 		protoc --proto_path=${_dir} --proto_path=${GOPATH}/src \
 			--proto_path=${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
+			--spireplugin_out=${_d} \
```

Comment:
hmm, i may want this to only run on plugin proto's... otherwise a bunch of codegen happens for things like api.